nnstreamer.ini.in

```diff
@@ -4,3 +4,6 @@ customfilters=@SUBPLUGIN_INSTALL_PREFIX@/customfilters/
 
 [decoder]
 decoders=@SUBPLUGIN_INSTALL_PREFIX@/decoders/
+
+[tensorflow]
```

Comment:
Add:
```
# Set 1 or TRUE if you want to eliminate memcpy of tensorflow input frames.
# It may break in some special cases (running tensorflow & nnstreamer in a chroot of a AWS VM); in such a case, keep it 0 or FALSE.
```